CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# CHANGELOG
+
+## [Unreleased]
+
+### Overview
+
+- Upgrades cosmos-sdk to `v0.47.3` LSM fork created by persistence, including [barberry](https://forum.cosmos.network/t/cosmos-sdk-security-advisory-barberry/10825) security fix
```

Comment:
Will need to specify FROM and TO versions, as it will imply some migration path. v45 -> v47 is a big leap, also IBC v4 -> v7 as well.

Reply:
done